### ISSUE TITLE
Documentation Fix:  can the documentation describe the commands needed when using Red Hats OCP platform.

### ISSUE THREAD

**Issue by Martin-M-Roberts** [IBM/kubernetes-driver#127]
The documentation needs to include a description of the commands that will be needed for an OCP deployment.  I recognise that these may be very similar to those of kubeadmin, but it should be made straight forward for OCP users.

**Comment by dvaccarosenna** [comment]
Agreed that the [deployment location](https://github.com/IBM/kubernetes-driver/blob/master/docs/user-guide/deployment-locations.md) section of the docs should include an example of obtaining config for OCP. From what I remember, the config you obtain through `oc` can often include a short lived token, which needs refreshing more frequently so this is not ideal for deployment locations.
